README.md

```diff
@@ -18,26 +17,62 @@ If you have any trouble, we will have facilitators on-site as early as 8:30 AM l
 
 As an alternative, a workshop session can be run on mybinder.org via this link: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/astropy/astropy-workshop/HEAD)
 
-## Schedule
-
-| Time (PT)     | Topic                                                          | Presenter/Instructor |
-|---------------|----------------------------------------------------------------|----------------------|
-| Before 9:00am | [Install and config](00-Install_and_Setup) help, if needed     | (All)         |
-| 9:00 - 9:20am | [Intro to Astropy and Code of Conduct](01-IntroCoC)            | Kelle Cruz ([@kelle](https://github.com/kelle))        |
-| 9:20 - 9:45am | [Astropy Units, Quantities, and Constants](03-UnitsQuantities) | Brett Morris ([@bmorris3](https://github.com/bmorris3))         |
-| 9:45 - 10:15am | [Intro to Object Oriented Programming (OOP)](02b-OOP)         | Brett Morris ([@bmorris3](https://github.com/bmorris3))    |
-| 10:15 - 10:30am | BREAK                                                          |                      |
-| 10:30 - 11:00am | [Coordinates](04-Coordinates)         |         |
-| 11:00 - 11:30pm | [Astropy Tables](06-Tables)                                    | Clara Brasseur ([@ceb8](https://github.com/ceb8))   |
-| 11:30 – 12:00pm | [I/O: FITS and ASCII](05-FITS)                                 | Clara Brasseur ([@ceb8](https://github.com/ceb8))       |
-| 12:00 - 1:30pm | LUNCH                                                          |                      |
-| 1:30 - 2:00pm | [Specutils](09b-Specutils)                                     | |
-| 2:00 - 2:30pm | [Photutils](09-Photutils)                                      |  |
-| 2:30 - 3:00pm | BREAK                                                            |                      |
-| 3:00 - 3:20pm | Explore together time                                                |     |
-| 3:20 - 3:40pm | [Uncertainty](13-Uncertainty)                              |     |
-| 3:40 - 4:00pm | [Astropy Communities & Contributing to Astropy](10-WrapUp)     |  |
-| 4:00 - 5:30pm | Explore together time                                          |           |
+### Review some introductory Python if you have not used Python
+
+That material is in the folder `02-PythonIntro`. A link to that -- so that you can read it even if you have not used Python before -- is [here](https://github.com/mwcraig/astropy-workshop/tree/ALOHA-2024/02-PythonIntro). Click on the individual notebooks, whose names end `.ipynb`, to open them.
+
+Doing this before the workshop will put you in a better position to follow along with the workshop materials.
+
+## Schedules
+
+At both sets of workshops there will be science use cases integrated into
+the presentation topics.
+
```

Comment:
```suggestion

Note that the materials are very similar between the two workshops, although the emphasis on the Subaru and Keck days are slightly different.

```

Reply:
Also remove the redundant "this is a duplicate" statement in individual entries.